src/roswire/ros2/state.py

```diff
@@ -107,7 +140,14 @@ def probe(self) -> ROS2SystemState:
                     mode = "serv"
                     continue
                 elif "Action Servers:" in line:
-                    break
+                    mode = "act_serv"
```

Comment:
Should this be `act_serv` or `a_serv`?

Reply:
Pulled these out as constants to avoid this

Reply:
Nice